ui/src/authors/components/AssignDrawer/__tests__/AssignDrawer.test.jsx

```diff
@@ -34,27 +37,21 @@ describe('AssignDrawer', () => {
     const onAssign = jest.fn();
     const selectedPapers = Set([1, 2, 3]);
 
-    const wrapper = shallow(
-      <AssignDrawer
-        visible={visible}
-        onDrawerClose={onDrawerClose}
-        onAssign={onAssign}
-        selectedPapers={selectedPapers}
-      />
+    const { getByTestId, getByRole } = render(
+      <Provider store={getStore()}>
+        <AssignDrawer
+          visible={visible}
+          onDrawerClose={onDrawerClose}
+          onAssign={onAssign}
+          selectedPapers={selectedPapers}
+        />
+      </Provider>
     );
-    expect(wrapper.find('[data-test-id="assign-button"]')).toHaveProp({
-      disabled: true,
-    });
-
-    wrapper
-      .find('[data-test-id="author-radio-group"]')
-      .simulate('change', { target: { value: 321 } });
-    wrapper.update();
-    expect(wrapper.find('[data-test-id="assign-button"]')).toHaveProp({
-      disabled: false,
-    });
+    expect(getByTestId('assign-button')).toBeDisabled();
+    getByRole('radio').click();
```

Comment:
here it might be better to explicitly check that the radio named "New author" is clicked, as this is what you testing in line 55.